database/README.md

```diff
@@ -26,34 +26,49 @@ Join our slack to discuss all things related to GoCryptoTrader! [GoCryptoTrader
 
 ## How to use
 
-##### To Manually migrate to the latest database you can run the "dbmigrate" helper in the cmd folder
+##### Prerequisites
 
-This will parse and run all migration files in your $GoCryptoTrader/database/migrations
 
-_This is also run from the bot when a connection is established to the database_
-
-```sh
-go run ./cmd/dbmigrate
+##### Create and Run migrations
```

Comment:
Before this add separate pathways for postgres and sqlite:
[ ] - Configuration examples
[ ] - Fetching of specific drivers 
[ ] - mention caveat for sqlite3 schema files (All lower case)